diff -up zathura-0.0.8.1/zathura.c.poppler15 zathura-0.0.8.1/zathura.c
--- zathura-0.0.8.1/zathura.c.poppler15	2010-10-06 16:35:44.966101001 -0400
+++ zathura-0.0.8.1/zathura.c	2010-10-06 16:35:58.124101001 -0400
@@ -4300,7 +4300,7 @@ cb_view_button_release(GtkWidget* widget
 
   /* get selected text */
   g_static_mutex_lock(&(Zathura.Lock.pdflib_lock));
-  char* selected_text = poppler_page_get_text(
+  char* selected_text = poppler_page_get_selected_text(
       Zathura.PDF.pages[Zathura.PDF.page_number]->page,SELECTION_STYLE,
       &rectangle);
 
